src/Tests/Microsoft.Diagnostics.Monitoring.EgressExtensibilityApp/Program.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.Tool.UnitTests;
+using Microsoft.Diagnostics.Tools.Monitor.Egress;
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.Text.Json;
+
+namespace Microsoft.Diagnostics.Monitoring.EgressExtensibilityApp
+{
+    internal sealed class Program
+    {
+        static int Main(string[] args)
+        {
+            RootCommand rootCommand = new RootCommand();
+
+            Command egressCmd = new Command("Egress");
+
+            egressCmd.SetHandler(() => Egress());
+
+            rootCommand.Add(egressCmd);
+
+            return rootCommand.Invoke(args);
+        }
+
+        private static int Egress()
+        {
+            EgressArtifactResult result = new();
+            try
+            {
+                string jsonConfig = Console.ReadLine();
+
+                ExtensionEgressPayload configPayload = JsonSerializer.Deserialize<ExtensionEgressPayload>(jsonConfig);
+                TestEgressProviderOptions options = BuildOptions(configPayload);
+
+                if (options.ShouldSucceed)
+                {
+                    result.Succeeded = true;
+                    result.ArtifactPath = EgressExtensibilityTests.SampleArtifactPath;
+                }
+                else
+                {
+                    result.Succeeded = false;
+                    result.FailureMessage = EgressExtensibilityTests.SampleFailureMessage;
+                }
+            }
+            catch (Exception ex)
+            {
+                result.Succeeded = false;
+                result.FailureMessage = ex.Message;
+            }
+
+            string jsonBlob = JsonSerializer.Serialize(result);
+            Console.Write(jsonBlob);
```

Comment:
```suggestion
            Console.WriteLine(jsonBlob);
```

By writing the newline, it's more clear to the consumer that the entire response was written.